CHANGELOG.md

```diff
@@ -19,11 +19,13 @@
 ### Fixed
 
 - Pass gitsync credentials through properly and use a fine-grained access token ([#489]).
+- BREAKING: The fields `connection` and `host` on `S3Connection` as well as `bucketName` on `S3Bucket`are now mandatory. Previously operators errored out in case these fields where missing ([#510]).
```

Comment:
I'd be in favor of removing the changelog entry, as this operator is not using S3 at all

Reply:
done